src/frontend/view/view-grid.ts

```diff
@@ -75,9 +75,46 @@ const createAgGrid = () => {
     ...(sortConfig[field] || {}),
   }))
 
+  const suppressKeyboardEvent = (
+    params: SuppressKeyboardEventParams
+  ): boolean => {
+    const KEY_TAB = 'Tab'
+    const key = params.event.key
+
+    if ([key, key.toUpperCase()].includes(KEY_TAB)) {
+      return true
+    }
+
+    return false
+  }
+
+  const suppressHeaderKeyboardEvent = (params): boolean => {
+    const KEY_TAB = 'Tab'
+    const key = params.event.key
+
+    const colNames = gridOptions.columnApi
+      .getAllDisplayedColumns()
+      .map((col) => col.colId)
+    const lastCol = colNames[colNames.length - 1]
+
+    if (
+      [key, key.toUpperCase()].includes(KEY_TAB) &&
```

Comment:
nit: as above, wouldn't this be equivalent?
```suggestion
      params.event.key.toUpperCase() === 'TAB' &&
```